### PR TITLE
Feature/ios premium

### PR DESCRIPTION
**Pull request by tomivm**
Support to purchase a subscription in the IOS Cordova App.

### CODE REVIEW

File: src/components/Settings/Subscribe/Subscribe.messages.js [hunk 1]
```diff
@@ -89,6 +89,11 @@ export default defineMessages({
     id: 'cboard.components.Settings.Subscribe.not_subscribed',
     defaultMessage: 'You are not subscribed. '
   },
+  unverified: {
+    id: 'cboard.components.Settings.Subscribe.unverified',
+    defaultMessage:
+      'An error ocurs during validation of your purchase. Please refresh'
```
Comment (RodriSanchez1) [inline src/components/Settings/Subscribe/Subscribe.messages.js]: 'An error occurred during validation of your purchase. Please refresh'

File: src/providers/SubscriptionProvider/SubscriptionProvider.actions.js [hunk 2]
```diff
@@ -214,7 +280,7 @@ export function updatePlans() {
           id: plan.planId,
           subscriptionId: plan.subscriptionId,
           billingPeriod: plan.period,
-          price: getPrice(plan.countries, locationCode),
+          price: getPrice(plan.countries, 'US'),
```
Comment (RodriSanchez1) [inline src/providers/SubscriptionProvider/SubscriptionProvider.actions.js]: I think you can add a 'US' default value on locationCode definition  instead.

File: src/translations/src/cboard.json [hunk 3]
```diff
@@ -484,6 +484,7 @@
   "cboard.components.Settings.Subscribe.paused": "Your subscription was paused on: {e}",
   "cboard.components.Settings.Subscribe.on_hold": "Your subscription expired on: {e}",
   "cboard.components.Settings.Subscribe.not_subscribed": "You are not subscribed. ",
+  "cboard.components.Settings.Subscribe.unverified": "An error ocurs during validation of your purchase. Please refresh",
```
Comment (RodriSanchez1) [inline src/translations/src/cboard.json]: See above comment

File: src/cordova-util.js [hunk 4]
```diff
@@ -91,6 +92,31 @@ const configAppPurchasePlugin = () => {
   store.initialize([Platform.GOOGLE_PLAY]);
 };
 
+const IOSconfigureInAppPurchasePlugin = () => {
+  const store = window.CdvPurchase.store;
+  const { ProductType, Platform } = window.CdvPurchase; // shortcuts
+  store.verbosity = 4;
```
Comment (RodriSanchez1) [inline src/cordova-util.js]: Delete this line!

**Review by martinbedouret — APPROVED** [review]

**Pull request merged**